api_types.ts

```diff
@@ -775,6 +776,11 @@ export interface ExecutionContext {
    * Information about state of the current sync. Only populated if this is a sync table formula.
    */
   readonly sync?: Sync;
+
+  /**
+   * {@link core.MetadataContext}
+   */
+  readonly parameters?: MetadataContext;
```

Comment:
As mentioned in another comment, `MetadataContext` doesn't support multiple sets of vararg arguments, which is not ideal. If we are going to introduce a new way to access arguments across all contexts, it would be great if it also resolved that issue.

Reply:
that's reasonable, although I'd prefer to do it in a separate followup. (this one's already had its scope increased from being only accessible by pack property options metadata formulas to all formulas)

Reply:
It's very hard to change the SDK surface in a breaking way later, but if we can tie it to the `parameters` => `arguments` change above then that would give us an out.